docs/upstreammonitoring.rst

```diff
@@ -0,0 +1,39 @@
+How to use rebase-helper for Upstream Monitoring systems
+========================================================
+
+The page describes an integration rebase-helper into an upstream monitoring systems.
```

Comment:
-> This page describes integration of rebase-helper into upstream monitoring systems.